src/components/atoms/SparkleLogoIcon/SparkleLogoIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import "./SparkleLogoIcon.scss";
+
+export const SparkleLogoIcon = () => (
+  <div className="SparkleLogoIcon">
+    <img
+      className="SparkleLogoIcon__logo"
+      src="/sparkle-logo-rf.svg"
```

Comment:
can you move the image to src/assets so that we could identify if we're missing it at compile time? 

and, please, rename the "-rf" suffix from the name.

Reply:
yep, done 5185f9a